### PR TITLE
Add confirm-ending article

### LINKED ISSUE #3: First article
### Repository

- [x] This user story was developed locally on a brach called `confirm-ending`.
- [x] The new branch was pushed to the shared repository
- [x] A pull request was made to `master` & reviewed by the team
- [x] Changes were merged to `master`

### HTML

- [x] article1 element that includes:
   - [x] header with the name of the city
   - [x] instruction paragraph
   - [x] two inputs for the arguments of the function
   - [x] button to execute the function
   - [x] paragraph with the result of the function

### CSS

- [x] Style of the article1

## JS
- [x] function endsWith with arguments linked to HTML input

### PR DESCRIPTION
**Pull request by aleks2407**
Closes gh-3

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -29,7 +29,75 @@ article {
...
 	width: 50%;
-	height: 150px;
+	height: 600px;
+}
+
+/* Article 1 */
+.article1 {
+	background-image: url(./img/alex/Moscow-Android-Wallpapers.jpg);
+	background-repeat: no-repeat;
+	background-size: 100%;
```
Comment (mametur) [inline style.css]: if you change here as  background-size: 100% 100%; then it will cover your box for all devices perfctly

File: style.css [hunk 2]
```diff
@@ -29,7 +29,75 @@ article {
 	margin-bottom: 10px;
 	background-color: aqua;
 	width: 50%;
-	height: 150px;
+	height: 600px;
```
Comment (mametur) [inline style.css]: we can have the same box for  nice UI as height :500px

File: scripts/confirm-ending.js [hunk 3]
```diff
@@ -62,14 +64,21 @@ function confirmEnding() { }
...
+  debugger;
   // core logic
-  const result = confirmEnding(str, ending);
+  const result = confirmEnding(arg1, arg2);
 
   // display for use
+  document.getElementById('result').innerHTML = result;
 
   // log for developers
+  console.log('Is "' + arg1 + '" ending with "' + arg2 + '" : ' + result);
```
Comment (mametur) [inline scripts/confirm-ending.js:79]: can you add more console logline as,
for instance, ` console.log('user first input', arg1, type: ${typeof arg1}` also for the rest. this is part of assignment

**Review by mametur — COMMENTED** [review]
@aleks2407  if you agree with my comments which I left on the lines then I will wait for your changes

**Review by oguzkarademir — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (aleks2407) [reply style.css]: yes, this works perfectly!thanks!

File: style.css (hunk 2 above)
Comment (aleks2407) [reply style.css]: i did push the box with the size of 500px. Don't get why it is 600px?

**update:** found it! I corrected it on article1, but not here. Done!

**Pull request merged**